openupgrade_scripts/scripts/website_crm/15.0.2.1/post-migration.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def migrate_website_crm_views(env):
+    """Migrate website_crm views in the post-migration process so that the new contact form
+    views have been defined.
+    This function checks if the "website_crm" module is installed. If it is, it looks for
+    the views associated with the website and updates the arch_db field replacing the
+    "mail.mail" occurrences with "crm.lead". This is done to adapt the
+    functionality in Odoo v15 where the default action is used to send an email and ensure
+    that it does not lose configuration to create an opportunity which in previous versions
+    was set by default by a replace in the form view."""
+    if env["ir.module.module"].search(
+        [("name", "=", "website_crm"), ("state", "=", "installed")]
+    ):
+        # Search views with associated website
+        website_views = env["ir.ui.view"].search(
```

Comment:
I think this is not the case. The problem is if there's no custom view, the default one (website_id=False) is doing another action.

Reply:
Done in the correct way so that it considers whether there is a customised view or not.